metadata-ingestion/src/datahub/ingestion/source/elastic_search.py

```diff
@@ -200,6 +204,12 @@ class ElasticProfiling(ConfigModel):
         default=False,
         description="Whether to enable profiling for the elastic search source.",
     )
+    operation_config: OperationConfig = Field(
+        default_factory=OperationConfig, description="To specify operation configs."
```

Comment:
Can we mark this as an experimental feature - don't want to provide backward compat guarantees on this given that we want to change it in the future

Reply:
Marked